src/main/scala/com/gu/datalakealerts/Features.scala

```diff
@@ -170,17 +155,13 @@ object Features {
   }
 
   case object BrazeBanner extends Feature {
-    override val id = "braze_banner"
+    val id = "braze_banner"
```

Comment:
Most members of [`Feature`](https://github.com/guardian/data-lake-alerts/blob/master/src/main/scala/com/gu/datalakealerts/Features.scala#L23-L28) are abstract (the type definition is provided but the concrete implementation is not), so they do not need to be explicitly overridden. `platformsToMonitor` has a concrete implementation and therefore the `override` is required (the compiler will complain if it is omitted). More info on this can be found [here](https://alvinalexander.com/scala/scala-trait-examples#abstract-and-concrete-fields-in-traits).

Reply:
I had wondered about this.. thanks for the explanation